### PR TITLE
TIP-880: split Behats by context

### PR DESCRIPTION
**Pull request by jjanvier**
This PR dispatches all the legacy Behats into proper "feature folders".
This has been done with @skeleton @LaurentPetard and @juliensnz. 

It's easier to review it if you pull the branch locally. Check the `tests/legacy` folder.

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | 
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

**Comment by anaelChardan** [comment]
You have 2 folders in for Product Models under enrichment, one is product-model and the other one is product_model

**Comment by jjanvier** [comment]
@anaelChardan fixed

**Comment by jjanvier** [comment]
@grena about `DBALPurger` and `MassEditJobContext` I didn't really moved them on purpose. I change the `Pim\Behat` folder to be able to have a `pim` directory for the features file. So I won't change that in this PR.

For the `channel/channel`, OK, I'll fix it.

**Comment by jjanvier** [comment]
@grena done

File: tests/legacy/features/pim/enrichment/product/pef/enabling.feature [hunk 1]
```diff
@@ -8,18 +8,14 @@ Feature: Enable and disable a product
     Given the "default" catalog configuration
     And I am logged in as "Julia"
 
-  @ce
   Scenario: Successfully disable a product
     Given an enabled "boat" product
     When I am on the "boat" product page
     And I disable the product
-    Then I should see the flash message "Product successfully updated"
     And product "boat" should be disabled
```
Comment (damien-carcel) [inline tests/legacy/features/pim/enrichment/product/pef/enabling.feature]: Then

File: tests/legacy/features/pim/enrichment/product/pef/enabling.feature [hunk 2]
```diff
@@ -8,18 +8,14 @@ Feature: Enable and disable a product
...
 
-  @ce
   Scenario: Successfully enable a product
     Given a disabled "boat" product
     When I am on the "boat" product page
     And I enable the product
-    Then I should see the flash message "Product successfully updated"
     And product "boat" should be enabled
```
Comment (damien-carcel) [inline tests/legacy/features/pim/enrichment/product/pef/enabling.feature]: Then

**Comment by jjanvier** [comment]
@damien-carcel fixed :)

**Pull request merged**